### ISSUE TITLE
Can't configure static IP config

### ISSUE THREAD

**Issue by micheljung** [meshtastic/web#111]
If you try this:
![image](https://github.com/meshtastic/web/assets/1864183/f8cb1037-9bfa-497a-8726-026778e17e48)

It silently fails like this:
![image](https://github.com/meshtastic/web/assets/1864183/e4e900da-9170-4c61-81d4-5578d44480e9)
`'invalid uint 32: string'` (for search purposes)

Because of this:
```JavaScript
/**
 * @generated from message meshtastic.Config.NetworkConfig.IpV4Config
 */
export declare class Config_NetworkConfig_IpV4Config extends Message<Config_NetworkConfig_IpV4Config> {
  /**
   *
   * Static IP address
   *
   * @generated from field: fixed32 ip = 1;
   */
  ip: number;

  /**
   *
   * Static gateway address
   *
   * @generated from field: fixed32 gateway = 2;
   */
  gateway: number;

  /**
   *
   * Static subnet mask
   *
   * @generated from field: fixed32 subnet = 3;
   */
  subnet: number;

  /**
   *
   * Static DNS server address
   *
   * @generated from field: fixed32 dns = 4;
   */
  dns: number;
```

**Comment by aptonline** [comment]
Seeing the same issue, unable to configure static ip settings under network. Hitting save does nothing.

**Comment by aptonline** [comment]
I've seen other posts saying you have to enter static details in reverse for this to work (1.1.168.192 for example) .  Not tested but will be trying it.

**Comment by aptonline** [comment]
That doesn't work for me, still not able to say settings for network (static IP etc) from WebUI. updated to latest beta firmware and same issue.

**Comment by aptonline** [comment]
Managed to set static IP via Android app but noticed that the IP details are encoded, is this correct?  See below.

![Screenshot from 2024-02-18 19-16-14](https://github.com/meshtastic/web/assets/190136/b8989c05-4309-44a1-b1e6-a486c0adbd0c)

**Comment by KomelT** [comment]
@Hunter275 this was fixed in #269.
